index.js

```diff
@@ -50,7 +60,16 @@ function getAllMovieTitles() {}
  *  checkIfAnyMovieHasRating(movies, "R");
  *  //> false
  */
-function checkIfAnyMovieHasRating() {}
+function checkIfAnyMovieHasRating(movies) {
```

Comment:
```suggestion
function checkIfAnyMovieHasRating(movies, rating = "G") {
```
You set one parameter, 'movies', but this function takes two parameters. The second one is a movie rating (ie "G", "PG", "PG-13", "R"). You can set a parameter to a default value using this syntax --> rating = "G"